python_modules/dagster/dagster/_core/storage/db_io_manager.py

```diff
@@ -70,9 +70,10 @@ def __init__(
         self,
         type_handlers: Sequence[DbTypeHandler],
         db_client: DbClient,
+        io_manager_name: Optional[str] = None,
     ):
         self._handlers_by_type: Dict[Optional[Type], DbTypeHandler] = {}
-        self._io_manager_name = self.__class__.__name__
+        self._io_manager_name = io_manager_name or self.__class__.__name__
```

Comment:
needed to add this back since we don't subclass DbIOManager for the duckdb and snowflake io managers. So we still need a way to customize the error messaging